packages/components/aside/src/aside.js

```diff
@@ -0,0 +1,76 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './aside.css';
+
+customElementDefineHelper(
+	'ts-aside',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		firstUpdated() {
+			this.shadowRoot.querySelector('slot[name="footer"]').addEventListener('slotchange', this.footerSlot.bind(this));
+		}
+
+		static get properties() {
+			return {
+				title: { type: String, attribute: 'data-title', reflect: true },
```

Comment:
Not sure about the reflect part! We only change the title for the empty title, but is it required to show it on the element as well? 🤔

Reply:
I am not sure, People may want to change the title and may be an empty title.